app/src/main/java/com/intellistart/interviewplanning/slot/application/port/NatsEventSubscriberInPort.kt

```diff
@@ -0,0 +1,9 @@
+package com.intellistart.interviewplanning.slot.application.port
+
+import com.intellistart.interviewplanning.output.pubsub.slot.SlotUpdatedEvent
+import reactor.core.publisher.Flux
+
+interface NatsEventSubscriberInPort {
```

Comment:
```suggestion
interface EventSubscriberInPort {
```

No infrastructure details in `*Port` interfaces.
Please, also provide what exactly this out port publishes.